ibis/omniscidb/client.py

```diff
@@ -20,7 +20,7 @@
 
 try:
     from cudf.dataframe.dataframe import DataFrame as GPUDataFrame
-except ImportError:
+except (ImportError, OSError):
```

Comment:
is there any way to add a test for this?

Reply:
Thanks @jreback for reviewing this PR.

sounds good I will open an issue on cudf repo! thanks!

I think that would be very hard to reproduce this error ib ibis CI, as cudf depends on CUDA .. it will be hard to reproduce this environment on  ibis CI.

as it is a very small fix, changing CI just for that would be not a good approach (also I am not sure that we can reproduce that on CI)

Reply:
https://github.com/rapidsai/cudf/issues/3956